scripts/Dockerfile.ecosystem

```diff
@@ -0,0 +1,14 @@
+# From the project root:
+# ```
+# cargo build
+# docker buildx build -f scripts/Dockerfile.ecosystem -t ruff-ecosystem-checker --load .
+# docker run --rm ruff-ecosystem-checker
+# ```
+
+FROM python:3.11
```

Comment:
What's your reasoning behind running it in a docker container vs running the script locally?

Reply:
https://moyix.blogspot.com/2022/09/someones-been-messing-with-my-subnormals.html or rather, running software not designed around working with arbitrary untrusted inputs with the largest set of untrusted inputs i could find